<launch>


    <param name="robot_description" textfile="$(find horizon_examples)/urdf/spot.urdf" />

    <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher" output="screen" >
        <param name="publish_frequency" type="double" value="250.0" />
    </node>

    <rosparam param = "foot_frames">['lf_foot', 'lh_foot', 'rf_foot', 'rh_foot']</rosparam>
    <param name="contact_model" value="1"/>
    <param name="number_of_legs" value="4"/>

    <rosparam param = "joint_init">[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0,
                                    0.0, 0.9, -1.52,
                                    0.0, 0.9, -1.52,
                                    0.0, 0.9, -1.52,
                                    0.0, 0.9, -1.52]</rosparam> <!-- 'leg_right_4_joint', 'leg_right_5_joint' -->

    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="dev" type="string" value="/dev/input/js0" />
        <!-- param name="autorepeat_rate" type="double" value="10." / -->
        <param name="deadzone" type="double" value="0.2" />
    </node>


    <!-- RVIZ -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find horizon_examples)/SRBD_horizon.rviz"/>


</launch>
